bool was_successful
duration time
int32 count
string arm
